packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -229,6 +229,36 @@ export default /* @ngInject */ function(
       });
   }
 
+  /**
+   * Get available release channels
+   * @returns {Promise}
+   */
+  function getAvailableReleaseChannels() {
+    self.releaseChannels = [];
+    return OvhApiOverTheBox.v6()
+      .getAvailableReleaseChannels({
+        serviceName: $stateParams.serviceName,
+      })
+      .$promise.then((channels) => {
+        channels.forEach((channel) => {
+          self.releaseChannels.push({
+            name: channel,
+            label: $translate.instant(`overTheBox_release_channel_${channel}`),
+          });
+        });
```

Comment:
`forEach` + `push` = `map`.
So you can: 
```suggestion
        self.releaseChannels = channels.map((channel) => ({
            name: channel,
            label: $translate.instant(`overTheBox_release_channel_${channel}`),
        }));
```